enterprise/backend/test/metabase_enterprise/advanced_permissions/api/setting_test.clj

```diff
@@ -2,64 +2,17 @@
   "Permisisons tests for API that needs to be enforced by Application Permissions to access Admin/Setting pages."
   (:require [clojure.test :refer :all]
             [metabase.api.geojson-test :as geojson-test]
-            [metabase.api.ldap-test :as ldap-test]
             [metabase.email :as email]
             [metabase.integrations.slack :as slack]
             [metabase.models :refer [Card Dashboard]]
             [metabase.models.permissions :as perms]
-            [metabase.models.setting-test :as models.setting-test]
             [metabase.public-settings.premium-features-test :as premium-features-test]
             [metabase.test :as mt]
-            [metabase.test.fixtures :as fixtures]
-            [metabase.test.integrations.ldap :as ldap.test])
+            [metabase.test.fixtures :as fixtures])
   (:import java.util.UUID))
 
 (use-fixtures :once (fixtures/initialize :db))
 
-(deftest setting-api-test
```

Comment:
Removed this test since it's no longer testing correct behavior, and the new behavior is tested in `metabase.api.setting-test`